content/docs/gto/command-reference/index.md

```diff
@@ -0,0 +1,19 @@
+# Using GTO Commands
```

Comment:
This is very early WIP. Many pages are just placeholders yet. Want to get feedback about the docs structure, suggested pages (even if they're empty - learn if page worth to be created), and Get Started page. cc @jorgeorpinel as always, and @francesco086 - you've been contributing a lot recently, and you're using GTO, so your feedback would be valuable here.

Reply:
> Want to get feedback about ... the docs structure

I don't think we need sections for use cases (there is one) or a user guide even, check https://dvc.org/doc/dvclive or even https://cml.dev/doc for example.

Focusing on Get Started for now.

Reply:
p.s. I think we should remove many TODO work from this PR so it's easier to review and faster to merge.

Reply:
Also, let's try to keep page titles consistent with URLs and nav entries:

```suggestion
# GTO Command Reference
```

> I just noticed this is an issue inherited from MLEM and [DVC docs](https://dvc.org/doc/command-reference)...